gulpfile.js

```diff
@@ -26,7 +26,7 @@ gulp.task('compile', [
 ])
 
 gulp.task('compile-bin', () => {
-  return gulp.src('bin/*')
+  return gulp.src('bin/**/*.js')
```

Comment:
I think `bin/next` and `next/next-build` wouldn't be compiled with this setting ?